Missing subject settings for emails

Some of the emails installed by the plugin don't have  a "subject" setting counterpart:
```
AssertionError: {'typesetting_proofreader_completed', 'typesetting_notify_typesetter', 'typesetting_proofreader_cancelled', 'typesetting_typesetter_deleted', 'typsetting_typesetter_decision_accept', 'typesetting_notify_proofreader', 'typsetting_typesetter_decision_decline', 'typesetting_complete', 'typesetting_typesetter_cancelled', 'accepted_review_reminder', 'typesetting_proofreader_reset', 'typesetting_typesetter_complete', 'from_address', 'default_review_reminder'} is not false : Found emails that don't have a subject setting
```
This can be tested from 5a1d96503a96a51108718d82770c066265b178ff by running core tests (`make command CMD="test core"`) with the typesetting plugin installed under `src/plugins/`

--- a/install/settings.json
+++ b/install/settings.json
@@ -7,11 +7,26 @@
       "description": "Email sent to Typesetter on task assignment.",
       "is_translatable": true,
       "name": "typesetting_notify_typesetter",
-      "pretty_name": "Typesetting Plugin: Typesetter Assignment Notification",
+      "pretty_name": "Typesetter Assignment Notification (PLUGIN)",
       "type": "rich-text"
     },
     "value": {
       "default": "<p>Dear {{ assignment.typesetter.full_name }},</p><p>This is an notification from {{ request.user.full_name }} that you have been assigned to the production of article {{ assignment.round.article.title }} on journal {{ request.journal.name }}. </p><p>You can view further information on this assignment <a href=\"{% journal_url 'typesetting_assignments' %}\" >here</a>.</p><p>Regards,</p><p>{{ request.user.signature|safe }}</p>"
+    }
+  },
+  {
+    "group": {
+      "name": "email_subject"
+    },
+    "setting": {
+      "description": "Subject for Typesetter Assignment Notification.",
+      "is_translatable": true,
+      "name": "subject_typesetting_notify_typesetter",
+      "pretty_name": "Subject for Typesetter Assignment Notification (PLUGIN)",
+      "type": "text"
+    },
+    "value": {
+      "default": "Typesetting Assignment"
     }
   },
   {
@@ -22,11 +37,26 @@
       "description": "Email sent to typesetting manager when a typesetter accepts a task.",
       "is_translatable": true,
       "name": "typsetting_typesetter_decision_accept",
-      "pretty_name": "Typesetting Plugin: Typesetter Accept",
+      "pretty_name": "Typesetter Accept (PLUGIN)",
       "type": "rich-text"
     },
     "value": {
       "default": "<p>Dear {{ assignment.manager.full_name }},</p><p>This is to notify you that {{ assignment.typesetter.full_name }} has accepted your assignment your assignment for {{ assignment.round.article.title|safe }}.</p>{% if note %}<p>They provided the following note: <br />{{ note|safe }} </p>{% endif %}<p>Regards,</p>"
+    }
+  },
+  {
+    "group": {
+      "name": "email_subject"
+    },
+    "setting": {
+      "description": "Subject for Typesetter Accept",
+      "is_translatable": true,
+      "name": "subject_typsetting_typesetter_decision_accept",
+      "pretty_name": "Subject for Typesetter Accept (PLUGIN)",
+      "type": "text"
+    },
+    "value": {
+      "default": "Typesetting Assignment Accepted"
     }
   },
   {
@@ -37,11 +67,26 @@
       "description": "Email sent to typesetting manager when a typesetter declines a task.",
       "is_translatable": true,
       "name": "typsetting_typesetter_decision_decline",
-      "pretty_name": "Typesetting Plugin: Typesetter Decline",
+      "pretty_name": "Typesetter Decline (PLUGIN)",
       "type": "rich-text"
     },
     "value": {
       "default": "<p>Dear {{ assignment.manager.full_name }},</p><p>This is to notify you that {{ assignment.typesetter.full_name }} has declined your assignment for {{ assignment.round.article.title|safe }}.</p>{% if note %}<p>They provided the following note: <br />{{ note|safe }} </p>{% endif %}<p>Regards,</p>"
+    }
+  },
+  {
+    "group": {
+      "name": "email_subject"
+    },
+    "setting": {
+      "description": "Subject for Typesetter Decline",
+      "is_translatable": true,
+      "name": "subject_typsetting_typesetter_decision_decline",
+      "pretty_name": "Subject for Typesetter Decline (PLUGIN)",
+      "type": "text"
+    },
+    "value": {
+      "default": "Typesetting Assignment Declined"
     }
   },
   {
@@ -52,11 +97,26 @@
       "description": "Email sent to typesetting manager when a typesetter completes a task.",
       "is_translatable": true,
       "name": "typesetting_typesetter_complete",
-      "pretty_name": "Typesetting Plugin: Typesetter Complete",
+      "pretty_name": "Typesetter Complete (PLUGIN)",
       "type": "rich-text"
     },
     "value": {
       "default": "<p>Dear {{ assignment.manager.full_name }},</p><p>This is to notify you that {{ assignment.typesetter.full_name }} has completed your assignment for {{ assignment.round.article.title|safe }}.</p>{% if note %}<p>They provided the following note: <br />{{ assignment.typesetter_note|safe }} </p>{% endif %}<p>Regards,</p>"
+    }
+  },
+  {
+    "group": {
+      "name": "email_subject"
+    },
+    "setting": {
+      "description": "Subject for Typesetter Complete  (PLUGIN)",
+      "is_translatable": true,
+      "name": "subject_typesetting_typesetter_complete",
+      "pretty_name": "Subject for Typesetter Complete (PLUGIN)",
+      "type": "text"
+    },
+    "value": {
+      "default": "Typesetting Assignment Complete"
     }
   },
   {
@@ -67,11 +127,26 @@
       "description": "Email sent to Proofreaders on task assignment.",
       "is_translatable": true,
       "name": "typesetting_notify_proofreader",
-      "pretty_name": "Typesetting Plugin: Proofreader Assignment Notification",
+      "pretty_name": "Proofreader Assignment Notification (PLUGIN)",
       "type": "rich-text"
     },
     "value": {
       "default": "<p>Dear {{ assignment.proofreader.full_name }},</p><p>This is an notification from {{ request.user.full_name }} that you have been assigned to proofread {{ assignment.round.article.title }} on journal {{ request.journal.name }}. </p><p>You can view further information on this assignment <a href=\"{% journal_url 'typesetting_assignments' %}\" >here</a>.</p><p>Regards,</p><p>{{ request.user.signature|safe }}</p>"
+    }
+  },
+  {
+    "group": {
+      "name": "email_subject"
+    },
+    "setting": {
+      "description": "Subject for Proofreader Assignment Notification",
+      "is_translatable": true,
+      "name": "subject_typesetting_notify_proofreader",
+      "pretty_name": "Subject for Proofreader Assignment Notification (PLUGIN)",
+      "type": "text"
+    },
+    "value": {
+      "default": "Proofreading Assignment"
     }
   },
   {
@@ -82,7 +157,7 @@
       "description": "Proofreader guidelines",
       "is_translatable": true,
       "name": "typesetting_proofreader_guidelines",
-      "pretty_name": "Typesetting Plugin: Proofreader Guide",
+      "pretty_name": "Proofreader Guide (PLUGIN)",
       "type": "rich-text"
     },
     "value": {
@@ -117,6 +192,21 @@
         "default": "<p>Dear editors,</p><p>This is an automatic notification to inform you that Typesetting is complete for article \"{{ article.title}}\".</p><p>This article is now in the <a href=\"{% url 'proofing_list' %}\">Proofing workflow</a>.</p><p>Regards,</p><p>{{ request.user.signature|safe }}</p>"
         }
   },
+  {
+    "group": {
+        "name": "email_subject"
+    },
+    "setting": {
+        "description": "Subject for Typesetting Complete (PLUGIN)",
+        "is_translatable": true,
+        "name": "subject_typesetting_complete",
+        "pretty_name": "Subject for Typesetting Complete (PLUGIN)",
+        "type": "text"
+    },
+    "value": {
+        "default": "Typesetting Complete"
+        }
+  },
 
   {
     "group": {
@@ -131,6 +221,22 @@
     },
     "value": {
         "default": "<p>Dear {{assignment.manager.full_name}},</p><p>This is a notification to let you know that the proofing you requested for the article {{assignment.round.article.title}}, with article number {{assignment.round.article.pk}}, has now been completed by {{assignment.proofreader.full_name}}.</p><p>Best wishes,</p>"
+        }
+  },
+
+  {
+    "group": {
+        "name": "email_subject"
+    },
+    "setting": {
+        "description": "Subject for Proofreading Complete (PLUGIN)",
+        "is_translatable": true,
+        "name": "subject_typesetting_proofreader_completed",
+        "pretty_name": "Subject for Proofreading Complete (PLUGIN)",
+        "type": "text"
+    },
+    "value": {
+        "default": "Proofreading Assignment Complete"
         }
   },
 
@@ -152,6 +258,22 @@
 
   {
     "group": {
+        "name": "email_subject"
+    },
+    "setting": {
+        "description": "Subject for Proofreading Cancelled (PLUGIN)",
+        "is_translatable": true,
+        "name": "subject_typesetting_proofreader_cancelled",
+        "pretty_name": "Subject for Proofreading Cancelled (PLUGIN)",
+        "type": "text"
+    },
+    "value": {
+        "default": "Proofreading Assignment Cancelled"
+        }
+  },
+
+  {
+    "group": {
         "name": "email"
     },
     "setting": {
@@ -163,6 +285,22 @@
     },
     "value": {
         "default": "<p>Dear {{assignment.proofreader.full_name}},</p><p>This is a notification to let you know that your proofreading request for the article {{assignment.round.article.title}}, with article number {{assignment.round.article.pk}}, has been reopened. Please log in to the journal using this link in order to complete this proofreading request: {% journal_url 'typesetting_proofreading_assignments' %}.</p><p>Thank you for your help,</p><p>Best wishes,</p><p>{{ assignment.manager.signature|safe }}</p>"
+        }
+  },
+
+  {
+    "group": {
+        "name": "email_subject"
+    },
+    "setting": {
+        "description": "Subject for Proofreading Reset (PLUGIN)",
+        "is_translatable": true,
+        "name": "subject_typesetting_proofreader_reset",
+        "pretty_name": "Subject for Proofreading Reset (PLUGIN)",
+        "type": "text"
+    },
+    "value": {
+        "default": "Proofreading Assignment Reopened"
         }
   },
 
@@ -184,6 +322,22 @@
 
   {
     "group": {
+        "name": "email_subject"
+    },
+    "setting": {
+        "description": "Subject for Typesetting Cancelled (PLUGIN)",
+        "is_translatable": true,
+        "name": "subject_typesetting_typesetter_cancelled",
+        "pretty_name": "Subject for Typesetting Cancelled (PLUGIN)",
+        "type": "text"
+    },
+    "value": {
+        "default": "Typesetting Assignment Cancelled"
+        }
+  },
+
+  {
+    "group": {
         "name": "email"
     },
     "setting": {
@@ -195,6 +349,22 @@
     },
     "value": {
         "default": "<p>Dear {{assignment.proofreader.full_name}},</p><p>This is a notification to let you know that your typesetting request for the article {{assignment.round.article.title}}, with article number {{assignment.round.article.pk}}, has been deleted. This means that you no longer need to complete this task.</p><p>Thank you for your help,</p><p>{{ assignment.manager.signature|safe }}</p>"
+        }
+  },
+
+  {
+    "group": {
+        "name": "email_subject"
+    },
+    "setting": {
+        "description": "Subject for Typesetting Deleted (PLUGIN)",
+        "is_translatable": true,
+        "name": "subject_typesetting_typesetter_deleted",
+        "pretty_name": "Subject for Typesetting Deleted (PLUGIN)",
+        "type": "text"
+    },
+    "value": {
+        "default": "Typesetting Assignment Cancelled"
         }
   }
 ]

